relay-event-schema/src/protocol/session.rs

```diff
@@ -28,6 +28,8 @@ pub enum SessionStatus {
     /// The session had an unhandled error, but did not crash.
     Unhandled,
     /// Unknown status, for forward compatibility.
+    ///
+    /// Adding a new variant still requires bumping the metrics extraction version.
```

Comment:
```suggestion
    /// If you add a new variant here, bump the session metrics extraction version to prevent outdated extraction in external relays.
```